spec/controllers/appointments_controller_spec.rb

```diff
@@ -63,6 +63,46 @@
       end
     end
 
+    describe "filtering by days overdue" do
+      it "displays patients only less than one year overdue when checked" do
+        really_overdue_appointment = create(:appointment,
+          facility: facility_2,
+          scheduled_date: 380.days.ago,
+          status: 'scheduled'
+        )
+        create(:blood_pressure, patient: really_overdue_appointment.patient, facility: facility_2)
+        really_overdue_patient_ids = really_overdue_appointment.patient_id
```

Comment:
There's only one, can we rename the var to be singular?

Reply:
Fixed.